test/relay_client_test.js

```diff
@@ -582,30 +596,32 @@ contract('RelayClient', function (accounts) {
   })
 
   describe('should handle incorrect relay hub contract in recipient', async function () {
-    let sr2
+    let gasSponsor2
     before(async function () {
+      const relayProvider = new RelayProvider(web3.currentProvider, relayClientConfig)
+      SampleRecipient.web3.setProvider(relayProvider)
       SampleRecipient.web3.currentProvider.relayOptions.isRelayEnabled = false
-      sr2 = await SampleRecipient.new()
+      gasSponsor2 = await TestEverythingAcceptedSponsor.new()
       // eslint-disable-next-line
       SampleRecipient.web3.currentProvider.relayOptions.isRelayEnabled = true
     })
 
     it('should revert on zero hub in recipient contract', async function () {
       try {
-        await sr2.emitMessage('hello world', { from: gasLess })
+        await sr.emitMessage('hello world', { from: gasLess, gasSponsor: gasSponsor2.address })
         assert.fail()
       } catch (error) {
-        assert.equal(true, error.message.includes('The relay hub address is set to zero in recipient at'))
+        assert.equal(true, error.message.includes('The relay hub address is set to zero in sponsor at'))
       }
     })
 
     it('should throw on invalid recipient', async function () {
       const tbk = new RelayClient(web3)
```

Comment:
haha. didn't know we still have tbk here..

Reply:
Will get rid of 'tbk' names of variables
###5